src/libcore/iter/traits.rs

```diff
@@ -386,10 +386,11 @@ pub trait Extend<A> {
 /// ```
 #[stable(feature = "rust1", since = "1.0.0")]
 pub trait DoubleEndedIterator: Iterator {
-    /// An iterator able to yield elements from both ends.
+    /// Removes and returns an element from the end of the iterator.
```

Comment:
Well it does remove an element. So does `next()` (from the beginning). Why
do you think this sounds wrong?

On Mon, Jul 11, 2016 at 11:16 AM, Guillaume Gomez <notifications@github.com>
wrote:

> In src/libcore/iter/traits.rs
> <https://github.com/rust-lang/rust/pull/34732#discussion_r70277691>:
>
> > @@ -386,10 +386,11 @@ pub trait Extend<A> {
> >  /// ```
> >  #[stable(feature = "rust1", since = "1.0.0")]
> >  pub trait DoubleEndedIterator: Iterator {
> > -    /// An iterator able to yield elements from both ends.
> > +    /// Removes and returns an element from the end of the iterator.
>
> Both sounds inaccurate to me. :-/
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rust-lang/rust/pull/34732/files/b4ff6b028ed6ae2e67d97604ad6bfbd235bb4f74#r70277691>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAC3nwp6zp2zpXgSnW_EAfiiwCdWX9-eks5qUl5ogaJpZM4JIh0J>
> .
>